pages/_/marketing-candidate/download.js

```diff
@@ -16,83 +16,60 @@ const STYLES_ROOT = css`
 `;
 
 const STYLES_H1 = css`
-  font-size: 1.953rem;
-  line-height: 1.25;
-  padding: 0px 0px 32px 0px;
+  font-size: ${Constants.typescale.lvl5};
+  line-height: 1.3;
+  padding: 0px 0px 24px 0px;
+  letter-spacing: -0.021rem;
   width: 100%;
-  color: ${Constants.system.pitchBlack};
-  @media (max-width: ${Constants.sizes.mobile}px) {
-    font-size: 1.953rem;
+  color: ${Constants.system.slate};
+  font-weight: 800;
```

Comment:
We can't use font-weight here, please use `font-family`.  Please change this to `font-weight: 400` and find the appropriately font-family you need for this, It should already be in `constants`.